_episodes/10-shielding.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "Live Coding Shielding Problem"
+teaching: 0
+exercises: 90
+questions:
+- "How can I perform a shielding calculation using Python?"
+keypoints:
```

Comment:
Regardless, these aren't the learning objectives for this lesson (though they are the activities). I think these are more along the lines of some Boom's Taxonomy LO's we want them to walk away with (in no particular order):
* Differentiate between data structures and choose appropriate structures for different cases
* value pair programming (this one may not be the case if we don't have them work in pairs)
* implement functions as a best practice
* implement for-loops
* produce a python script from scratch
* execute python scripts in a command line

Reply:
oh wait, just saw keypoints at the bottom of the page. I think those are good. Though the learning objectives should still be added to the top.

Reply:
I modified these a bit. Open to alterations. Lemme know what you think.

Reply:
So a few comments on what you wrote: Words like "Understand" and "Learn" are not actually good words for learning outcomes (in fact "learn" is not an outcome) since they are so broad and vague so actually stating the action we expect them to be able to do with them. Also, they should all be actions since it is "what do we want students to be able to do?" See my comments below regarding a few specific lines. They may seem nitpicky, but this is good practice for just general lesson development and being specific about what the take aways are so that we can guide our teaching.